gapic/src/main/com/google/gapid/GraphicsTraceView.java

```diff
@@ -370,13 +371,14 @@ private static void flatten(
 
       Framebuffer(View.Framebuffer, "Framebuffer", DefaultPosition.Center, FramebufferView::new),
       Pipeline(View.Pipeline, "Pipeline", DefaultPosition.Center, PipelineView::new),
-      Textures(View.Textures, "Textures", DefaultPosition.Center, TextureView::new),
+      TextureList(View.TextureList, "Textures", DefaultPosition.Center, TextureList::new),
       Geometry(View.Geometry, "Geometry", DefaultPosition.Center, GeometryView::new),
       Shaders(View.Shaders, "Shaders", DefaultPosition.Center, ShaderView::new),
       Performance(View.Performance, "Performance(Experimental)", DefaultPosition.Center, PerformanceView::new),
       Report(View.Report, "Report", DefaultPosition.Center, ReportView::new),
       Log(View.Log, "Log", DefaultPosition.Center, (p, m, w) -> new LogView(p, w)),
 
+      TextureView(View.TextureView, "Texture", DefaultPosition.Right, TextureView::new),
```

Comment:
nit: The names of these two tabs seem to be too close?

Reply:
Yes, I was considering to rename the "Textures" tab to "Texture List", but I preferred to keep the former to avoid the additional space requirement in the tab bar if we consistently would add "List" to all resource list tabs. It might not be a problem if users prefer to have the texture list and the views in separate UI folders.